### PR TITLE
docs: admin-api.md document adds configuration https example

### PR DESCRIPTION
**Pull request by Firstsawyou**
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by spacewander — APPROVED** [review]

**Comment by Firstsawyou** [comment]
@moonming  Please have a look when you are free.

File: docs/en/latest/admin-api.md [hunk 1]
```diff
@@ -571,12 +571,16 @@ Config Example:
     "key": "",
     "name": "upstream-for-test",
     "desc": "hello world",
+    "scheme":"http",            # The scheme used when communicating with upstream, the default is `http`
```
Comment (juzhiyuan) [inline docs/en/latest/admin-api.md:574]:
```suggestion
    "scheme": "http",            # The scheme used when communicating with upstream, the default is `http`
```

File: docs/zh/latest/admin-api.md [hunk 2]
```diff
@@ -578,15 +578,18 @@ upstream 对象 json 配置内容：
     "checks": {},               # 配置健康检查的参数
     "hash_on": "",
     "key": "",
-    "name": "upstream-xxx",      # upstream 名称
+    "name": "upstream-xxx",     # upstream 名称
     "desc": "hello world",      # upstream 描述
+    "scheme":"http"             # 跟上游通信时使用的 scheme，默认是 `http`
```
Comment (juzhiyuan) [inline docs/zh/latest/admin-api.md:583]: ditto

File: docs/en/latest/admin-api.md [hunk 3]
```diff
@@ -657,12 +659,32 @@ After the execution is successful, nodes will not retain the original data, and
 
 ```
 
+Example 2: Create upstream and configure the scheme as `https`.
```
Comment (moonming) [inline docs/en/latest/admin-api.md:662]: This is an example of proxying to upstream https

File: docs/en/latest/admin-api.md [hunk 4]
```diff
@@ -657,12 +659,32 @@ After the execution is successful, nodes will not retain the original data, and
...
+
+```shell
+# Create upstream and configure the scheme as `https`
```
Comment (moonming) [inline docs/en/latest/admin-api.md:665]: We need a complete example, from apisix configuration to client access

**Review by moonming — CHANGES_REQUESTED** [review]

**Comment by tokers** [comment]
I cannot capture whether the https is used in client and admin api or the apisix to upstream.

File: docs/en/latest/admin-api.md (hunk 4 above)
Comment (moonming) [reply docs/en/latest/admin-api.md:665]: @Firstsawyou any update？

File: docs/zh/latest/admin-api.md (hunk 2 above)
Comment (Firstsawyou) [reply docs/zh/latest/admin-api.md]: updated.

File: docs/en/latest/admin-api.md (hunk 3 above)
Comment (Firstsawyou) [reply docs/en/latest/admin-api.md]: updated.

File: docs/en/latest/admin-api.md (hunk 1 above)
Comment (Firstsawyou) [reply docs/en/latest/admin-api.md]: updated.

File: docs/en/latest/admin-api.md [hunk 5]
```diff
@@ -657,12 +659,44 @@ After the execution is successful, nodes will not retain the original data, and
 
 ```
 
+Example 2: How to proxy upstream https.
```
Comment (tokers) [inline docs/en/latest/admin-api.md:662]: Bad title, I think "How to proxy requests to upstream with HTTPS scheme" is better.

File: docs/zh/latest/admin-api.md [hunk 6]
```diff
@@ -665,11 +668,42 @@ HTTP/1.1 200 OK
 
 ```
 
+示例二：如何代理上游 https。
+
+1、创建 route 并 配置 upstream 的 scheme 为 `https`。
```
Comment (tokers) [inline docs/zh/latest/admin-api.md:673]: The empty space between `并` and `配置` is redundant.

File: docs/zh/latest/admin-api.md [hunk 7]
```diff
@@ -665,11 +668,42 @@ HTTP/1.1 200 OK
...
+    }
+}'
+```
+
+执行成功后，请求与上游通信时的 scheme 将为 `https`。
+
+2、 发送请求进行测试。
+
+```shell
+$ curl http://127.0.0.1:9080/get -i
```
Comment (tokers) [inline docs/zh/latest/admin-api.md:694]: IMHO, this test case lacks persuasion. You'd better also print the result returned by the upstream.

**Review by tokers — APPROVED** [review]

**Comment by Firstsawyou** [comment]
@moonming Please take a look when you have time.

File: docs/en/latest/admin-api.md [hunk 8]
```diff
@@ -571,12 +571,16 @@ Config Example:
...
     "name": "upstream-for-test",
     "desc": "hello world",
+    "scheme": "http",           # The scheme used when communicating with upstream, the default is `http`
 }
 ```
 
-Example：
+**Example:**
+
+Example 1: Create an upstream and modify the corresponding data.
```
Comment (moonming) [inline docs/en/latest/admin-api.md:580]: modify what data? please desc more detail

File: docs/en/latest/admin-api.md [hunk 9]
```diff
@@ -657,12 +659,54 @@ After the execution is successful, nodes will not retain the original data, and
 
 ```
 
+Example 2: How to proxy requests to upstream with HTTPS scheme.
```
Comment (moonming) [inline docs/en/latest/admin-api.md:662]: Example 2: How to proxy client request to `https` upstream service

File: docs/zh/latest/admin-api.md [hunk 10]
```diff
@@ -665,11 +668,52 @@ HTTP/1.1 200 OK
 
 ```
 
+示例二：如何使用 HTTPS 的 scheme 将请求代理到上游。
```
Comment (moonming) [inline docs/zh/latest/admin-api.md:671]: 将客户端请求代理到上游https服务

**Review by moonming — CHANGES_REQUESTED** [review]

File: docs/zh/latest/admin-api.md (hunk 10 above)
Comment (Firstsawyou) [reply docs/zh/latest/admin-api.md]: Updated.

File: docs/en/latest/admin-api.md (hunk 9 above)
Comment (Firstsawyou) [reply docs/en/latest/admin-api.md]: Updated.

File: docs/en/latest/admin-api.md (hunk 8 above)
Comment (Firstsawyou) [reply docs/en/latest/admin-api.md]: Updated.

File: docs/en/latest/admin-api.md (hunk 4 above)
Comment (moonming) [reply docs/en/latest/admin-api.md]: Please check other desc too.

Comment (Firstsawyou) [reply docs/en/latest/admin-api.md]: Okay, I have checked for updates.

**Comment by Firstsawyou** [comment]
@moonming Please take a look when you have time.

**Review by moonming — APPROVED** [review]

**Pull request merged**